client/components/site-selector/style.scss

```diff
@@ -29,6 +29,21 @@
 	&.is-large .site-selector__sites {
 		border-top: 1px solid var(--color-neutral-10);
 	}
+
+	.search-component {
+		.search-component__input {
+			width: 100%;
+		}
+
+		.components-button {
+			border: none;
+			box-shadow: none;
+	
+			&:hover {
+				background: transparent;
+			}
```

Comment:
This is for Jetpack Cloud. Jetpack Cloud has some broad global styles for button components. They don't look correct on the site selector search component, so I've declared here that these buttons (the magnifying glass and X icons) should not have borders or background hover colors.